internal/service/common/utils/utils.go

```diff
@@ -174,3 +174,21 @@ func CompareObjects[T db.Model](a, b T, excluded ...string) DBTag {
 
 	return tags
 }
+
+// GetColumns gets the DB column names for the given fields
+func GetColumns[T db.Model](record T, fields []string) []string {
```

Comment:
perhaps something like this func (r DBTag) Columns() []any  line 15 but for string

Reply:
yeah the problem is that DbTags is a map which is not ordered. This way we can make sure the tags are coming out inorder

from 
```
tags := utils.GetDBTagsFromStructFields(dbModel, "AlarmDictionaryVersion", "EntityType", "Vendor", "ObjectTypeID")
columns := []string{tags["AlarmDictionaryVersion"], tags["EntityType"], tags["Vendor"], tags["ObjectTypeID"]}
```
to
```
columns := utils.GetColumns(dbModel, []string{"AlarmDictionaryVersion", "EntityType", "Vendor", "ObjectTypeID"})
```